src/i_video.c

```diff
@@ -1794,7 +1790,8 @@ void I_InitGraphics(void)
     {
         I_Error("Failed to initialize video: %s", SDL_GetError());
     }
-    SDL_SetHint("SDL_HINT_RENDER_BATCHING", "0");
+
+    SDL_SetHint(SDL_HINT_RENDER_DRIVER, "direct3d11");
```

Comment:
The last time I checked, direct3d11 still had frame pacing problems under certain configurations that outweigh the benefits. Let me do some testing later to see if that's still the case.

Reply:
I tested it on my machine and it seems to work as direct3d9. I had forgotten how we used to find frame pacing issues.

Reply:
If you have a typical 60Hz fixed-refresh display (probably the majority of people without a vrr display), and use the default settings, there's odd stuttering with direct3d11:

![1](https://github.com/fabiangreffrath/woof/assets/56656010/ad7471b1-25d6-48d1-b3c8-341a4ea30039)
![2](https://github.com/fabiangreffrath/woof/assets/56656010/20a32404-2df9-4d90-b0f0-65a3e1403c4e)

The same thing happens with other settings, sometimes worse, but that's a quick example.

Reply:
I think last time we experimented with this, the consensus quickly became "let SDL figure out the best renderer itself, we cannot know all the corner cases". I think screenshots broke with a specific renderer at that time.

Reply:
Okay, I removed it here: https://github.com/fabiangreffrath/woof/pull/1755/commits/781859806842b7d97431dde08359567e223a0a3f